dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultMetadataImportService.java

```diff
@@ -223,7 +224,7 @@ public MetadataImportParams getParamsFromMap( Map<String, List<String>> paramete
         {
             User overrideUser = null;
 
-            if ( parameters.containsKey( "overrideUser" ) )
+            if ( Optional.ofNullable( parameters ).filter( p -> p.containsKey( "overrideUser" ) ).isPresent() )
```

Comment:
Is `parameters` ever null? Our code can be pretty defensive at times. I often found the solution is to make sure collections are non-null by default.